### PR TITLE
Add mention in README about binary data and base64 support

### PR DESCRIPTION
**Pull request by migueleliasweb**

### CODE REVIEW

**Comment by migueleliasweb** [comment]
Ping :bell: @jekkel

File: README.md [hunk 1]
```diff
@@ -33,6 +33,8 @@ Both are identical multi-arch images built for `amd64`, `arm64`, `arm/v7`, `ppc6
 - Update/Delete on change of configmap or secret
 - Enforce unique filenames
 - CI tests for k8s v1.19-v1.26
+- Support for base64'd `binaryData` for both `Secret` and `ConfigMap` kinds
+  - Binary data content is base64 decoded before generating the file
```
Comment (jekkel) [inline README.md:37]: Could you add a word about the recent fix in #250 ? So one now can have a URL in `binaryData` and the http response as stored binary as well then.

**Review by jekkel — APPROVED** [review]

**Review by jekkel — APPROVED** [review]

**Comment by migueleliasweb** [comment]
Improved the wording. Let me know if you think it's good enough for now. :+1:

File: README.md [hunk 2]
```diff
@@ -33,6 +33,10 @@ Both are identical multi-arch images built for `amd64`, `arm64`, `arm/v7`, `ppc6
 - Update/Delete on change of configmap or secret
 - Enforce unique filenames
 - CI tests for k8s v1.19-v1.26
+- Support `binaryData` for both `Secret` and `ConfigMap` kinds
+  - Binary data content is base64 decoded before generating the file on disk
+  - Values can also be base64 encoded URLs that download binary data e.g. executables
+    - The value in the `ConfigMap`/`Secret` must end with "`.url`" ([see](https://github.com/svolland-csgroup/k8s-sidecar/blob/master/test/resources/resources.yaml#L11))
```
Comment (jekkel) [inline README.md:39]: Nice! But I'm afraid you need to rebase this branch and
* point the link to [line 84](https://github.com/kiwigrid/k8s-sidecar/blob/master/test/resources/resources.yaml#L84) rather then 11
* change the wording to point out that the **key** of the configmap or secret entry needs to end on `.url`, not the value.

Comment (migueleliasweb) [reply README.md]: Fixed 🤞

**Review by jekkel — APPROVED** [review]

**Comment by migueleliasweb** [comment]
🎉

**Pull request merged**